src/main/java/org/jilt/internal/AbstractBuilderGenerator.java

```diff
@@ -331,12 +331,18 @@ protected final TypeVariableName mangleTypeParameter(TypeVariableName typeVariab
     }
 
     private ParameterSpec setterParameterSpec(VariableElement attribute, TypeName parameterType) {
-        ParameterSpec.Builder ret = ParameterSpec.builder(parameterType,
-                this.attributeSimpleName(attribute));
-        AnnotationMirror nullableAnnotation = this.firstAnnotationCalledNullable(attribute);
+        ParameterSpec.Builder ret = ParameterSpec.builder(parameterType, this.attributeSimpleName(attribute));
+        AnnotationMirror nullableAnnotation = firstAnnotationCalledNullable(attribute);
         if (nullableAnnotation != null) {
             ret.addAnnotation(AnnotationSpec.get(nullableAnnotation));
         }
+
+        for (AnnotationMirror annotation : attribute.getAnnotationMirrors()) {
+            if (!annotation.equals(nullableAnnotation)) {
+                ret.addAnnotation(AnnotationSpec.get(annotation));
+            }
+        }
```

Comment:
It doesn't really make much sense to copy `@Nullable` annotations separately if we're copying all annotations anyway.

Let's unify the two parts:

```suggestion
        for (AnnotationMirror annotation : attribute.getAnnotationMirrors()) {
            ret.addAnnotation(AnnotationSpec.get(annotation));
        }
```

Reply:
You are absolutely right about this